.whiskey/action_hooks/pre-build

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
```

Comment:
It is a pre-build hook which adds the source of this repository to requirements.txt. Basically as doing ``pip install -e ./`` Previous build instructions pull from github rather than building from source. If the user chooses the automated builds, that will still happen, but this way it is possible to build from source locally for development. I will expand the comments here to indicate the file's purpose.